library/Fission/User/Creator/Class.hs

```diff
@@ -42,19 +43,22 @@ class Heroku.AddOn.Creator m => Creator m where
     -> m (Either Errors UserId)
 
 instance MonadIO m => Creator (Transaction m) where
-  create username password email herokuAddOnId now =
-    Password.hashPassword password >>= \case
+  create username didOrPW email herokuAddOnId now =
```

Comment:
Just cause I think my other comment got loss:
Still not sure if this is smart or a ridiculous :thinking: 
The thinking is, anytime a user is created, they will _always_ have _either_ a did _or_ a password

Reply:
Hmmm yeah. It really doesn’t help us when we what to say, know who has what type of auth (to tell to upgrade), and could potentially lead to confusing edge cases. We want to be able to make solid assumptions about the data in our application that’s passed validation.

How would you feel about changing this class to have 2 methods: creatWithPassword and createWithDID? We can later extend that if we add more types of keys.

Reply:
Also createWithDID doesn’t need to know about Heroku, which is a plus IMO 😛